server/utils/placementRequests/placementSummaryList.test.ts

```diff
@@ -0,0 +1,94 @@
+import { TextItem } from '@approved-premises/ui'
+import { placementRequestDetailFactory } from '../../testutils/factories'
+import { placementSummaryList } from './placementSummaryList'
+import { DateFormats } from '../dateUtils'
+import { detailedStatus, statusTextMap } from '../placements'
+import cas1SpaceBookingSummaryFactory from '../../testutils/factories/cas1SpaceBookingSummary'
+
+describe('placement summary list', () => {
+  it('renders a summary list for a placement request with an upcoming space booking', () => {
+    const spaceBooking = cas1SpaceBookingSummaryFactory.upcoming().build()
+    const placementRequestDetail = placementRequestDetailFactory.withSpaceBooking(spaceBooking).build()
+
+    expect(placementSummaryList(placementRequestDetail)).toEqual({
+      rows: [
+        { key: { text: 'Approved Premises' }, value: { text: spaceBooking.premises.name } },
+        {
+          key: { text: 'Date of match' },
+          value: { text: DateFormats.isoDateToUIDate(placementRequestDetail.booking.createdAt) },
+        },
+        {
+          key: { text: 'Expected arrival date' },
+          value: { text: DateFormats.isoDateToUIDate(spaceBooking.expectedArrivalDate) },
+        },
+        {
+          key: { text: 'Expected departure date' },
+          value: { text: DateFormats.isoDateToUIDate(spaceBooking.expectedDepartureDate) },
+        },
+        { key: { text: 'Status' }, value: { text: statusTextMap[detailedStatus(spaceBooking)] } },
+        { key: { text: 'Delius event number' }, value: { text: spaceBooking.deliusEventNumber } },
+      ],
+    })
+  })
+
+  it('renders the actual arrival date for a current booking', () => {
+    const spaceBooking = cas1SpaceBookingSummaryFactory.current().build()
+    const placementRequestDetail = placementRequestDetailFactory.withSpaceBooking(spaceBooking).build()
+
+    const result = placementSummaryList(placementRequestDetail)
+
+    expect(result.rows).toEqual(
+      expect.arrayContaining([
+        {
+          key: { text: 'Actual arrival date' },
+          value: { text: DateFormats.isoDateToUIDate(spaceBooking.actualArrivalDate) },
+        },
+      ]),
+    )
+    expect(result.rows).not.toEqual(
```

Comment:
I'm wondering why we don't show 'Expected arrival date' if there is an actual. For the CRU, I would have thought that they only really care about the expected - the booked dates. When the person actually turned up is more the concern of the AP.

Reply:
I think that's a very good point -- this was my call, not in designs, so I'll reinstate the 'expected' dates and only add the actual ones if available -- this should work?